string pid
string type
float64[] pose